serverless/admin_guide/serverless-ossm-setup.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-setup"]
+= Integrating Service Mesh with OpenShift Serverless
+:context: serverless-ossm-setup
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Using {ProductShortName} with {ServerlessProductName} enables developers to configure additional networking and routing options.
+
+The {ServerlessOperatorName} provides Kourier as the default ingress for Knative. However, you can use {ProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. The following procedures provide instructions for integrating {ProductShortName} with or without Kourier enabled.
+
+[IMPORTANT]
+====
+{ServerlessProductName} only supports the use of {ProductName} functionality that is explicitly documented in this guide, and does not support other undocumented features.
+====
+
+// without kourier
+[id="serverless-ossm-setup-native"]
+== Integrating {ProductShortName} with {ServerlessProductName} natively
+
+Integrating {ProductShortName} with {ServerlessProductName} natively, without Kourier, allows you to use additional networking and routing options that are not supported by the default Kourier ingress, such as mTLS functionality.
```

Comment:
_allows you to use additional networking..._
Should this sentence be in the intro paragraphs above the heading, so folks know why they would want to do one method over the other?

Reply:
What would we put then under this heading though? 😄 I don't want to duplicate the info but we need an intro paragraph for this too?

Reply:
_However, you can use {ProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. Integrating with Kourier disabled allows you to configure additional networking and routing options that the Kourier ingress does not support._ 
Something like that?

Reply:
Great suggestion, thank you!